MediaAdminBundle/ExtractReference/ExtractReferenceInterface.php

```diff
@@ -33,6 +40,15 @@ public function extractReference(StatusableInterface $statusableElement);
     public function getReferencePattern($statusableElementId);
 
     /**
+     * Get invalidate tag of statusable element for reference
+     *
+     * @param string $reference
+     *
+     * @return string|null
+     */
+    public function getInvalidateTagStatusableElement($reference);
```

Comment:
@alavieille : the purpose of this method is not clear, could you rename it into something more relevant ? What is an invalidate tag, is it simply a cache tag ?